app/Resources/EzSystemsFormBuilderBundle/views/fields/captcha/basic.html.twig

```diff
@@ -1,18 +1,27 @@
 {% extends 'EzSystemsFormBuilderBundle:fields:field.html.twig' %}
-{% trans_default_domain "formcaptcha" %}
+{% trans_default_domain "formfield" %}
 
 {% set imageId = 'captcha__' ~ field.id|replace({'-': '_'}) %}
 {% set captchaImage = ez_form_captcha_path(field) %}
 
 {% block input %}
-    <div class="row">
-        <div class="col-sm-4 col-xs-12">
-            <img class="ezform-captcha-image" id="{{ imageId }}" src="{{ captchaImage }}" alt="" />
-            <p><a class="ezform-captcha-reload btn btn-default btn-sm" href="#">{{ 'formcaptcha.reload_image'|trans }}</a></p>
-        </div>
-        <div class="col-sm-8 col-xs-12">
-            <input type="text" class="ezform-field ezform-captcha form-control" id="{{ field.id }}" placeholder="{{ field.placeholderText }}">
-        </div>
+    <div class="ezform-field-group" data-error-message="{{ 'field.is.required'|trans }}">
+        <span class="ezform-field-indicator"></span>
+        <span class="ezform-field-wrapper">
+            <input
+                id="{{ field.id }}"
+                type="text"
+                class="ezform-field ezform-captcha form-control"
+                placeholder="{{ field.placeholderText }}"
+                {% if field.isRequired %}required{% endif %}
+                >
```

Comment:
@andrerom of course. I thought it's more readable this way, when a tag has a lot of attributes inside.